policies/travel-policy.md

```diff
@@ -251,48 +284,44 @@ and travel arrangements.
 Reimbursements
 --------------
 
-The Traveller must keep all receipts, tickets etc related to the
-reimbursement and submit these with the **Expenses Reimbursement Form**.
-Photos or scanned versions of the receipts or credit card statements are
-acceptable. In the event a receipt is not available or lost the
-Traveller must submit a signed statement detailing the item/s purchased
-and the cost.
-
-The Traveller is expected to submit an **Expenses Reimbursement Form**
+Except where noted below, the traveller must keep all receipts, tickets
+etc related to the reimbursement and submit these with the [Expenses
+Reimbursement Form]. Photos or scanned versions of the receipts or credit
+card statements are acceptable. In the event a receipt is not available or
+lost the Traveller must submit a signed statement detailing the item/s
+purchased and the cost.
+
+One currency should be used consistently within each Expense Reimbursement Request.
+ie if claiming Euro's the entire request should be in Euro's.
+Where an expense is incurred in a currency other than the main currency of the Expense 
+Reimbursement request, it will need to be converted to the main currency before being 
+entered into the form.
+
+Currency Conversion:
+-   Where the expense is incurred on a credit card, debit card or via a bank transfer or 
+    bank payment use the amount actually charged to you by the relevant financial institution.
+-   Otherwise use the the rate you were charged (if available) or the exchange rate on the day 
+    the expense was incurred and note this exchange rate in the description field on the [Expenses Reimbursement Form].
```

Comment:
If you're not paying by card/bank transfer the presumably that means you are paying cash. In that case I have previously used the exchange rate I was charged when I converted the currency.

Reply:
I've updated the conversion requirements accordingly.